terraform/modules/paas/resources.tf

```diff
@@ -0,0 +1,61 @@
+resource cloudfoundry_app web_app {
+  name                       = local.web_app_name
+  space                      = data.cloudfoundry_space.space.id
+  health_check_type          = "http"
+  health_check_http_endpoint = "/ping"
+  instances                  = var.web_app_instances
+  memory                     = var.web_app_memory
+  docker_image               = var.docker_image
+  strategy                   = "blue-green-v2"
+  timeout                    = 180
+  environment                = var.app_environment_variables
+
+  service_binding {
+    service_instance = cloudfoundry_service_instance.postgres.id
+  }
+  service_binding {
+    service_instance = cloudfoundry_service_instance.redis.id
+  }
+  routes {
+    route = cloudfoundry_route.web_app_cloudapps_digital_route.id
+  }
+}
+
+resource cloudfoundry_app worker_app {
+  name              = local.worker_app_name
+  space             = data.cloudfoundry_space.space.id
+  health_check_type = "port"
+  instances         = var.worker_app_instances
+  memory            = var.worker_app_memory
+  docker_image      = var.docker_image
+  strategy          = "blue-green-v2"
+  timeout           = 180
+  environment       = var.app_environment_variables
+
+  service_binding {
+    service_instance = cloudfoundry_service_instance.postgres.id
+  }
+  service_binding {
+    service_instance = cloudfoundry_service_instance.redis.id
+  }
+}
+
+resource cloudfoundry_route web_app_cloudapps_digital_route {
+  domain   = data.cloudfoundry_domain.local.id
+  space    = data.cloudfoundry_space.space.id
+  hostname = local.web_app_name
+}
+
+resource cloudfoundry_service_instance postgres {
+  name         = local.postgres_service_name
+  space        = data.cloudfoundry_space.space.id
+  service_plan = data.cloudfoundry_service.postgres.service_plans["small-11"]
+  json_params  = "{\"enable_extensions\": [\"pg_buffercache\",\"pg_stat_statements\", \"plpgsql\", \"btree_gin\", \"btree_gist\"]}"
```

Comment:
Can we confirm these extensions are required?
 @gpeng @misaka maybe?

Reply:
@saliceti this is the current list
|       Name        | Version |   Schema   |                        Description                        |
--------------------|---------|------------|-----------------------------------------------------------|
| btree_gin          | 1.0     | public     | support for indexing common datatypes in GIN |
| btree_gist         | 1.2     | public     | support for indexing common datatypes in GiST |
| pg_buffercache     | 1.2     | public     | examine the shared buffer cache |
| pg_stat_statements | 1.4     | public     | track execution statistics of all SQL statements executed |
| plpgsql            | 1.0     | pg_catalog | PL/pgSQL procedural language |

Reply:
We're definitely using the btree stuff. I think we have a use case for `uuid-ossp` by the look of it. There's a card in this sprint to start generating UUIDs on records and it may be useful.

Reply:
Thanks. I would remove plpgsql at least. We don't use stored procedures and hopefully we never will.

Reply:
removed `plpgsql` extension

Reply:
`uuid-ossp` can we add that in please, 

There is dev work will involves GUID/UUID that will throw PaaS back into the dark ages.


```
select uuid_generate_v4() 
```

Without it we will be forced to generate uuid in `ruby` land and we know how rubbish that is.

Reply:
appears `uuid-ossp` exists by default, the above query also returns `UUID`